src/plugins/data/common/search/aggs/buckets/lib/ip_prefix.ts

```diff
@@ -0,0 +1,19 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+export interface IpPrefixAggKey {
+  type: 'ip_prefix';
+  address: string;
+  prefix_length: number;
+}
+
+export type IpPrefixKey = IpPrefixAggKey;
+
+export const convertIPPrefixToString = (range: IpPrefixKey, format: (val: any) => string) => {
```

Comment:
range doesnt seem as the correct variable name here

Reply:
Thanks for the feedback! I spent some time debating this more after your comment. Since the variable is of type "IpPrefixKey" which is basically an IP Address (e.g. 192.168.0.0) coupled with a Numeric Prefix (e.g /16) -> so 192.168.0.0/16 - I guess "cidr" would be a more appropriate name than range. This naming then ends up mirroring the naming in search/aggs/buckets/lib/cidr_mask.ts which also handles strings of the form "address/prefix" (e.g. 192.168.0.0/16).

I've updated the use to cidr here and in the next comment.